nnoir-onnx/nnoir_onnx/operators/prelu.py

```diff
@@ -0,0 +1,20 @@
+from nnoir.functions import *
+from .utils import *
+
+
+class OpPRelu(Op):
+
+    def __init__(self, node):
+        super(OpPRelu, self).__init__(node)
+
+    def to_function(self, env, constants):
+        [x, slope] = self.node.input
+
+        if slope in constants.keys():
+            c = constants[slope].ravel()
+            if len(c) != 1:
```

Comment:
Check it
```
>>> import numpy as np
>>> x = np.empty((1,2,3))
>>> len(x)
1
>>> x.size
6
```

Reply:
What do you mean?
`c` is already `ravel`ed, so the shape is expected one-dimension.
Then, `len(x) and x.size` should be the same.

Reply:
I understand if you say we should reduce the number of method calls (then we should use `x.size` rather than `len(x)`).

Reply:
Sorry, I missed already `ravel` ed.